src/plugins/dashboard/public/application/test_helpers/make_default_services.ts

```diff
@@ -35,9 +35,11 @@ export function makeDefaultServices(): DashboardAppServices {
     for (let i = 0; i < size; i++) {
       hits.push({
         id: `dashboard${i}`,
-        title: `dashboard${i} - ${search} - title`,
-        description: `dashboard${i} desc`,
-      });
+        attributes: {
+          title: `dashboard${i} - ${search} - title`,
+          description: `dashboard${i} desc`,
+        },
+      } as any);
```

Comment:
Another `as any`. As soon as I've added typings a few problems appeared. It might be good to revise TS types across the "dashboard" app

<img width="1066" alt="Screenshot 2022-07-21 at 16 38 08" src="https://user-images.githubusercontent.com/2854616/180254955-4e3d9bfa-e18d-4dee-96b2-e4c1e15dbce0.png">